### PR TITLE
Handle NULL defaultLocale (bug 1132488)

### PR DESCRIPTION
**Pull request by kumar303**
This traceback was introduced by in-app JWTs
in bug 1131708

### CODE REVIEW

File: webpay/pay/forms.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def clean_req(self):
             raise forms.ValidationError(msg.BAD_ICON_KEY)
 
         for fn in msg.SHORT_FIELDS:
-            if (len(payload['request'].get(fn, '')) >
+            if (len(payload['request'].get(fn, None) or '') >
```
Comment (muffinresearch) [inline webpay/pay/forms.py]: I found this hard to parse for a moment - but I see now. A comment might be handy although the test will avoid this getting accidentally "factored" out. :)

**Comment by muffinresearch** [comment]
r+wc

File: webpay/pay/forms.py (hunk 1 above)
Comment (kumar303) [inline webpay/pay/forms.py]: hopefully the test will keep it from getting refectored out ;) but yeah I'll add a note. This is a typical string problem in Python - original code was sloppy.
